engine/language-server/src/main/scala/org/enso/languageserver/filemanager/Path.scala

```diff
@@ -28,6 +28,12 @@ case class Path(rootId: UUID, segments: Vector[String]) {
     val parentDir = toFile(rootPath)
     new File(parentDir, fileName)
   }
+
+  def startsWith(path: Path): Boolean = {
```

Comment:
The name is self-descriptive, I know, but we try to write docs for all public methods and classes.